examples/keystore-management/README.md

```diff
@@ -4,42 +4,52 @@ A simple Next.js application to manage Waku RLN keystores.
 
 ## Overview
 
-This application provides an interface for managing keystores for Waku's rate-limiting nullifier (RLN) functionality. It integrates with MetaMask for wallet connectivity and demonstrates how to work with the Waku RLN library.
+This application provides an interface for managing keystores for Waku's rate-limiting nullifier (RLN) functionality. It integrates with MetaMask for wallet connectivity.
 
 ## Features
 
 - Connect to MetaMask wallet
 - View wallet information including address, network, and balance
-- Support for Sepolia testnet
+- Support for Linea Sepolia testnet only
 - Keystore management functionality
+- Token approval for RLN membership registration
 
 ## Getting Started
 
 1. First, install the dependencies:
 
 ```bash
 npm install
-# or
-yarn
 ```
 
 2. Run the development server:
 
 ```bash
 npm run dev
-# or
-yarn dev
 ```
 
 3. Open [http://localhost:3000](http://localhost:3000) with your browser.
 
-4. Connect your MetaMask wallet (Sepolia testnet is supported).
+4. Connect your MetaMask wallet (Linea Sepolia testnet is required).
 
-## Technologies
+## Linea Sepolia Network
+
+This application is configured to use ONLY the Linea Sepolia testnet. If you don't have Linea Sepolia configured in your MetaMask, the application will help you add it with the following details:
+
+- **Network Name**: Linea Sepolia Testnet
+- **RPC URL**: https://rpc.sepolia.linea.build
+- **Chain ID**: 59141
+- **Currency Symbol**: ETH
+- **Block Explorer URL**: https://sepolia.lineascan.build
+
+You can get Linea Sepolia testnet ETH from the [Linea Faucet](https://faucet.goerli.linea.build/).
```

Comment:
The link to the faucet doesn't work for me. FWIW I already had Sepolia ETH and was able to bridge using https://bridge.linea.build/